package-res/resources/web/prompting/components/StaticAutocompleteBoxComponent.js

```diff
@@ -54,7 +54,7 @@
  * @class
  * @extends BaseComponent
  */
-define([ 'common-ui/util/util', 'cdf/components/BaseComponent', 'dojo/number', 'amd!cdf/lib/jquery.ui' ], function(Utils, BaseComponent, DojoNumber,
+define([ 'common-ui/util/util', 'cdf/components/BaseComponent', 'dojo/i18n', 'dojo/number', 'amd!cdf/lib/jquery.ui' ], function(Utils, BaseComponent, DojoNumber,
```

Comment:
you added a new argument to the require list but no argument at the function. No regressions with this/jquery.ui variable?